<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>config_dialog</class>
 <widget class="QDialog" name="config_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>환경설정</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>일반</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>새로고침 주기(초)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_refresh_time">
           <property name="minimum">
            <number>15</number>
           </property>
           <property name="maximum">
            <number>120</number>
           </property>
           <property name="singleStep">
            <number>5</number>
           </property>
           <property name="value">
            <number>30</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_payment_request_load">
         <property name="text">
          <string>지급신청 로드</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_auto_show_ai_results">
         <property name="text">
          <string>AI 결과 자동 띄우기</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>단축키</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_layout">
      <attribute name="title">
       <string>레이아웃</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_layout_main">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_layout_containers">
         <item>
          <widget class="QGroupBox" name="groupBox_left_container">
           <property name="title">
            <string>왼쪽 영역</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_left">
            <item>
             <widget class="QListWidget" name="listWidget_layout_left">
              <property name="dragEnabled">
               <bool>true</bool>
              </property>
              <property name="acceptDrops">
               <bool>true</bool>
              </property>
              <property name="dragDropMode">
               <enum>QAbstractItemView::DragDrop</enum>
              </property>
              <property name="defaultDropAction">
               <enum>Qt::MoveAction</enum>
              </property>
              <item>
               <property name="text">
                <string>썸네일</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_layout_content">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_content">
            <item>
             <widget class="QLabel" name="label_content">
              <property name="text">
               <string>메인 컨텐츠
(고정)</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_right_container">
           <property name="title">
            <string>오른쪽 영역</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_right">
            <item>
             <widget class="QListWidget" name="listWidget_layout_right">
              <property name="dragEnabled">
               <bool>true</bool>
              </property>
              <property name="acceptDrops">
               <bool>true</bool>
              </property>
              <property name="dragDropMode">
               <enum>QAbstractItemView::DragDrop</enum>
              </property>
              <property name="defaultDropAction">
               <enum>Qt::MoveAction</enum>
              </property>
              <item>
               <property name="text">
                <string>정보 패널</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_info_header">
         <property name="text">
          <string>정보 패널 내부 순서 (상세)</string>
         </property>
         <property name="styleSheet">
          <string>font-weight: bold; margin-top: 5px;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="listWidget_info_order">
         <property name="styleSheet">
          <string>QListWidget::item {
    background-color: #2b2b2b;
    color: #e0e0e0;
    border: 1px solid #444444;
    border-radius: 4px;
    margin: 2px;
    padding: 8px;
}
QListWidget::item:selected {
    background-color: #1a4a4a;
    border: 1px solid #008080;
    color: white;
}</string>
         </property>
         <property name="dragDropMode">
          <enum>QAbstractItemView::InternalMove</enum>
         </property>
         <property name="defaultDropAction">
          <enum>Qt::MoveAction</enum>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <item>
          <property name="text">
           <string>이메일</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>메모관리</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>서류미비 및 확인필요</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>DA 추가요청(수신)</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>config_dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>config_dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
